projects/canopy/src/lib/forms/sort-code/sort-code.component.spec.ts

```diff
@@ -33,12 +34,13 @@ const errorStateMatcherMock = mock(LgErrorStateMatcher);
 
 @Component({
   template: `
-    <form (ngSubmit)="submit()" [formGroup]="form" #testForm="ngForm">
```

Comment:
the ngSubmit is redundant and the submit method creates errors (as it doesn't exist) when submitting the form (in the unit tests)